.devcontainer/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       REDIS_SERVER_HOST: redis
 
   db:
-    image: postgres:latest
+    image: pgvector/pgvector:pg17
```

Comment:
Why is the version here `pg17`, while the version in `compose.yml.example` is `pg16`?